bundle/manifests/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -439,6 +439,7 @@ spec:
   provider:
     name: Medik8s
     url: https://www.medik8s.io/
+  replaces: self-node-remediation.v0.0.0
```

Comment:
I'm wondering what happens when the referred version doesn't exist, this might cause trouble in CI? 🤔  We will see...

Reply:
looks ok I guess :man_shrugging: